### PR TITLE
Jetpack Manage: Fix a style glitch introduced on /dashboard view

### PR DESCRIPTION
**Pull request by cleacos**
## Proposed Changes

This PR fixes a style issue introduced on the `/dashboard` section with this PR https://github.com/Automattic/wp-calypso/pull/87814:

image

I've removed the `+` icon and touched the styles to show the `Issue License` text.

In `/sites` after this fix:

image

---
Anyway, we had to remove that `+` based on the A4A designs:

image

When we move to A4A, we will have to exchange the button colors and change `Issue License` to `Add products`

## Testing Instructions

- Check on `/dashboard` that the `+` icon doesn't appear
- Check on `/sites`, by clicking on any site on the list, that you see the button `Issue License` instead of the `+`

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-top-header-buttons/index.tsx
Comment (kallehauge) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-top-header-buttons/index.tsx]: `span.sites-overview__issue-license-button-caption` isn't used anywhere (you can try and remove it in the browser as well), but I assume it had a function in context of being next to the icon.
Should we perhaps remove it as well, if we remove the icon?

**Review by kallehauge — APPROVED** [review]
Work as expected 🚀 

I'm approving so we can ship it ASAP considering it looks much better than production, but we should get final approval from design that we're removing the icon in Jetpack Manage (not just the new dashboard)

(_We have approval to make the dashboard look more like A4A, but I don't think we have carte blanche to do it for other areas in Jetpack Manage_ 😄)

**Comment by cleacos** [comment]
> we should get final approval from design that we're removing the icon in Jetpack Manage (not just the new dashboard)

The icon didn't exist in Jetpack Manage previously. It was introduced today by the new `sites` section.

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-top-header-buttons/index.tsx
Comment (cleacos) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-top-header-buttons/index.tsx]: `span` wrapping element removed. It wasn't there before. 2730ef2d8f27bd73581aae9f58e7398c64e89539

**Pull request merged**
